### ISSUE TITLE
Visualization of training views

### ISSUE THREAD

**Issue by wangyuyy** [sarafridov/K-Planes#23]
I found that in videodataset, train dataset returns a batchsize of rays while test dataset returns rays of a whole image.
But I want to check out the reconstruction quality of the training views(not just mse or PSNR informations), when the training was done.
I tried to modify the `__getitem__` method of the videodataset by ignoring the `if self.split == 'train'`, but the output is still not correct.
So I wonder if there is an easy way to achieve it.

**Comment by Giodiro** [comment]
I think the easiest way would be to use (part of) the training set as validation set, so you can get plots for the training images.
Basically change the data-loading to load the training set twice instead of train+test

**Comment by zhenyuan1234** [comment]
I have a problem. How to output a rendered image instead of a video using a trained model. How to set it up while testing.
